test/e2e-tests-deposit.sh

```diff
@@ -9,7 +9,7 @@ truffle exec scripts/setup_environment.js
 truffle exec scripts/deposit.js 3 3 18 
 truffle exec scripts/mine_blocks.js 21
 
-sleep 5
+sleep 10
```

Comment:
We should figure out a better way here than waiting a fixed amount of time. Maybe we can check repeatedly and fail if 10s have passed and it’s still not there.

Reply:
Perhaps, but the driver sleep for 5 seconds anyway. So repeatedly trying will never succeed until at least 5 seconds has passed.

Reply:
Theoretically the driver could pick up the change 1ms after the 21 blocks have been mined (if its personal 5s had passed in that moment). That means we might be waiting 10s too long in the worst case. 

Having these sleeps in tests is generally an anti pattern which we should try to solve more elegantly than just increasing the time to wait.